### PR TITLE
Update submodules

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

**Review by sdroege — APPROVED** [review]

File: gtk/src/auto/notebook.rs [hunk 1]
```diff
@@ -527,6 +527,9 @@ pub trait NotebookExt: 'static {
     #[doc(alias = "gtk_notebook_set_action_widget")]
     fn set_action_widget(&self, widget: &impl IsA<Widget>, pack_type: PackType);
 
+    #[doc(alias = "gtk_notebook_set_current_page")]
+    fn set_current_page(&self, page_num: i32);
```
Comment (sdroege) [inline gtk/src/auto/notebook.rs:531]: This is manually implemented: https://gtk-rs.org/gtk3-rs/git/docs/src/gtk/notebook.rs.html#415

**Comment by sdroege** [comment]
@GuillaumeGomez You can update gir another time now

**Comment by GuillaumeGomez** [comment]
Updated!

**Review by sdroege — APPROVED** [review]

**Comment by sdroege** [comment]
You could remove `#[allow(clippy::let_and_return)]`

**Comment by GuillaumeGomez** [comment]
Good idea. Done as well.

**Comment by GuillaumeGomez** [comment]
Ah, there is still one missing in here too.

cc @filnet

**Comment by filnet** [comment]
> Ah, there is still one missing in here too.

I'll see what I can do. In the meantime I suggest to revert the related change.

**Comment by sdroege** [comment]
Yeah and then let's get this in for now.

**Comment by GuillaumeGomez** [comment]
Removed the last commit then.

**Pull request merged**